docs/src/user-guide/guides-using-object-storage/compress.md

```
@@ -0,0 +1,78 @@
+# Compressing logs
+
+To compress logs from S3, you'll need to:
+
+1. Enable the CLP IAM user to access the S3 path containing your logs.
+2. Use the `s3` subcommand of `sbin/compress.sh` to compress your logs.
+
+## IAM user configuration
+
+Attach the following policy to the CLP IAM user by following [this guide][add-iam-policy].
```

Markdownlint (0.37.0)

Missing link or image reference definition: "add-iam-policy"
Reference links and images should use a label that is defined

(MD052, reference-links-images)